html/components/autocomplete.stories.js

```diff
@@ -0,0 +1,210 @@
+import { useEffect } from '@storybook/client-api';
+import { autocomplete } from './autocomplete';
+import { markdownDocumentationLinkBuilder } from '../../storybook-utilities/markdownDocumentationLinkBuilder';
+
+export default {
+  title: 'Components/Autocomplete',
+  decorators: [(story) => `<div class="sprk-o-Box">${story()}</div>`],
+  parameters: {
+    info: `
+${markdownDocumentationLinkBuilder('autocomplete')}
+- Live demos of the Autocomplete are available on our [demo site](https://spark-testhtml.netlify.app/autocomplete.html).
+
+#### Keyboard Interactions
+- The list can be closed by clicking on the document body or by pressing
+the Escape key.
+- When the list is open, items in the list can be highlighted using the
+Up and Down Arrow keys. Note that this highlight is distinct from the
+browser focus, which stays in the input.
+
+#### Accessibility
+- The component should include an element with \`aria-live="polite"\`. The
+  contents of this element should be updated to reflect the number of visible
+  items in the list.
+- The input should have \`aria-autocomplete="list"\` and \`role="combobox"\`.
+  These attributes correctly identify the control to Assistive Technology.
+- The input should have \`autocomplete="off"\`, \`autocapitalize="off"\`,
+  and \`spellcheck="false"\`. These attributes deactivate default browser
+  behavior that can interfere with autocomplete behavior.
+- When the list is hidden, the text input should have \`aria-expanded="false"\`.
+- When the list is visible, the text input should have \`aria-expanded="true"\`.
+- When the list is visible and an item is highlighted (with the arrow keys),
+  the text input should have \`aria-activedescendant\` set to the \`id\`
+  of the active item.
+- When a list item is highlighted (with the arrow keys), it should have
+  \`aria-selected="true"\`.
+- Each selectable list item should have \`role="option"\`.
+
+
+#### Requires Additional Engineering:
+In order to keep the Spark Autocomplete generic enough to use in a wide
```

Comment:
It's small but I feel like the word 'generic' almost feels like a negative thing in this context. Maybe the word 'flexible' would be better? Not a hill I'm going to die on.

Reply:
I like flexible as well!